package.json

```diff
@@ -13,11 +13,13 @@
   },
   "dependencies": {
     "argon2": "^0.29.1",
+    "bootstrap": "^5.2.1",
     "dotenv": "^16.0.2",
     "express": "^4.18.1",
     "jsonwebtoken": "^8.5.1",
     "pg": "^8.8.0",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.5.0",
```

Comment:
if you are not using them you can uninstall them

```shell
npm uninstall bootstrap react-bootstrap
```

Reply:
<img width="763" alt="Screen Shot 2022-09-13 at 5 35 28 PM" src="https://user-images.githubusercontent.com/106217355/190033069-a6f349fd-a95b-4f1e-a4c6-740c15728371.png">